spec/Powertrain/Transmission.vspec

```diff
@@ -70,12 +70,19 @@ TravelledDistance:
 #
 # Current gear
 #
-Gear:
+CurrentGear:
```

Comment:
Here we touch the "deprecated" topic. If we know or can assume that no-one is using the old signal, then I think it is fine that we just remove/rename the existing signal. If not we should keep the old signal as well and mark it as deprecated, to give users time to change to the new signal name